### PR TITLE
Developer3

### PR DESCRIPTION
**Pull request by Tirumala12345**

### CODE REVIEW

File: java [hunk 1]
```diff
@@ -1,6 +1,5 @@
 Java Full Stack
-Angular
-Html css
-Spring Boot
 Core Java
+Angular
```
Comment (chandrakala8899) [inline java:3]: Pls add proper comments

Comment (Tirumala12345) [reply java:3]: Okay Sure I will update

Comment (chandrakala8899) [reply java:3]: Pls change  Naming Conventions

Comment (Tirumala12345) [reply java:3]: Okay Sure

**Review by chandrakala8899 — APPROVED** [review]

**Pull request merged**